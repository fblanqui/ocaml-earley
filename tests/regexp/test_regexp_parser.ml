open Earley_core.Regexp

let _ =
  let (re, _) = regexp_from_string Sys.argv.(1) in
  Printf.printf "%S â‡’ %a\n%!" Sys.argv.(1) print_regexp re
