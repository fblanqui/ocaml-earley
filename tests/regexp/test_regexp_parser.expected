"ab?c*" ⇒ Seq([Chr('a');Opt(Chr('b'));Str(Chr('c'))])
"[ab]+" ⇒ Pls(Set(ab))
"a*" ⇒ Str(Chr('a'))
